dev/docker/hive/start.sh

```diff
@@ -39,6 +39,12 @@ cp -f ${HADOOP_TMP_CONF_DIR}/* ${HADOOP_CONF_DIR}
 cp -f ${HIVE_TMP_CONF_DIR}/* ${HIVE_CONF_DIR}
 sed -i "s/__REPLACE__HOST_NAME/$(hostname)/g" ${HADOOP_CONF_DIR}/core-site.xml
 sed -i "s/__REPLACE__HOST_NAME/$(hostname)/g" ${HADOOP_CONF_DIR}/hdfs-site.xml
+
+if [[ -n "${ENABLE_JDBC_AUTHORIZATION}" ]]; then
```

Comment:
Change `ENABLE_JDBC_AUTHORIZATION` to `ENABLE_SQLBASE_AUTHORIZATION `?

anothe questiong, Will `ENABLE_JDBC_AUTHORIZATION` and `RANGER_HIVE_REPOSITORY_NAME, xxx` conflict??

Reply:
Yes, we they should choose one mode of them.

Reply:
I added a check for it. You can see.